features/donation.feature

```diff
@@ -153,6 +153,7 @@ Feature: The goal is to donate one time or multiple time with a subscription
     When I click on the "1" "img" element
     Then I should see "Votre soutien compte beaucoup pour moi."
 
+#    Check if I can't continue create a new subscription and then can cancel a subscription
```

Comment:
I'm don't really agree with this comment indentation. Whitespaces are useless here. 

Please juste write `# Check if I can't continue... ` indented with the `Given I am on "/don"`